rust/blockstore/src/arrow/provider.rs

```diff
@@ -79,6 +79,16 @@ impl ArrowBlockfileProvider {
         Ok(BlockfileWriter::ArrowBlockfileWriter(file))
     }
 
+    pub fn purge_all_entries(&self) {
+        while let Some((key, _)) = self.block_manager.block_cache.pop() {
```

Comment:
similar approach to the hnsw cache purge since foyer doesn't have a remove/purge all method